modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/representative_form_upload_controller.rb

```diff
@@ -13,6 +13,7 @@ class RepresentativeFormUploadController < ApplicationController
         upload_scanned_form
         upload_supporting_documents
       ]
+      before_action :deny_access_unless_686c_enabled, only: [:submit]
```

Comment:
should apply to all routes currently in this controller. can remove the `before_action` options if you want (it would apply to all routes by default).